src/main/java/com/aws/greengrass/deployment/DeviceConfiguration.java

```diff
@@ -345,6 +346,9 @@ void initializeComponentStore(String nucleusComponentName, Semver componentVersi
         Permissions.setArtifactPermission(destinationArtifactPath, FileSystemPermission.builder()
                 .ownerRead(true).ownerExecute(true).groupRead(true).groupExecute(true)
                 .otherRead(true).otherExecute(true).build());
+        // Relink the alts path to point to the artifact since we've just installed. This will allow the customer to
+        // delete their unzipped Nucleus distribution.
+        kernelAlts.relinkInitLaunchDir(destinationArtifactPath);
```

Comment:
Why does this need to be called in two places? Both `initializeComponentStore` or `setupInitLaunchDirIfAbsent` get invoked during nucleus launch

Reply:
The alts directory is initially setup in the constructor of KernelAlternatives and the initial value points to the directory it is launched from because that happens earlier than this call and because this call will not always happen.
This change is needed so that if we start and we install to the component store (which is not a given) then we will correct the alts directory. If we are not installing or we cannot find the launch dir as expected then the behavior is unchanged.